src/main/java/com/github/manolo8/darkbot/core/manager/GuiManager.java

```diff
@@ -153,6 +154,8 @@ public GuiManager(Main main, PluginAPI pluginAPI, RepairManager repairManager) {
         this.assembly = register("assembly");
 
         register("ggBuilder", GateSpinnerGui.class);
+        register("refinement_count");
```

Comment:
does this have any relationship with this pr?

Reply:
I added this one-row change of registration GUI window is need for getting window  position for click accept updating generators with promerium. It not have relationship with diminish quest but it's useful in plugin code because provide required information via API.
I mean it allow exclude reflection code, and implement updating logic on not-patched bot.
![image](https://github.com/darkbot-reloaded/DarkBot/assets/52008918/e3f36487-3986-4dca-9693-7015bc8b9ab7)